autounattend.xml

```diff
@@ -0,0 +1,4568 @@
+<?xml version="1.0" encoding="utf-8"?>
+<unattend xmlns="urn:schemas-microsoft-com:unattend" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State">
+  <!-- UnattendedWinstall https://github.com/memstechtips/UnattendedWinstall -->
+  <settings pass="offlineServicing"></settings>
+  <settings pass="windowsPE">
+    <component name="Microsoft-Windows-Setup" processorArchitecture="x86" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
+      <UserData>
+        <ProductKey>
+          <Key>VK7JG-NPHTM-C97JM-9MPGT-3V66T</Key>
```

Comment:
I'm just checking with you, this is not a real cdkey, is it?

Reply:
Hey @Brzhk thanks for your concern, but it's not a leak.
Microsoft provides keys that determine which edition of Windows will be installed depending on the key used, but Windows can't be activated with that key.
In this case, it's the Pro edition of Windows, you can find more info at the links below.

https://learn.microsoft.com/en-us/windows-server/get-started/kms-client-activation-keys?tabs=server2025%2Cwindows1110ltsc%2Cversion1803%2Cwindows81

https://gist.github.com/rvrsh3ll/0810c6ed60e44cf7932e4fbae25880df